### PR TITLE
Feat refactor hero section #723

### PR DESCRIPTION
**Pull request by jibola2588**
## Related Issue

refactor hero code with issue number #723 

## Description

The hero code was refactored to avoid the repetition of codes

## Screenshots

Screenshot (87)

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/project-add.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 name: 🌱 Project Addition
 description: Add a new project to the list
 title: "data: project addition by "
-labels: ["🌱 chore: projects addition", "👍  status: approved", "gssoc23", "level1"]
+labels: ["🌱 chore: projects addition", "👍  status: approved", "gssoc23", "🟨 level 1"]
```
Comment (priyankarpal) [inline .github/ISSUE_TEMPLATE/project-add.yml:4]:
``` suggestion
labels: ["🌱 chore: projects addition", "👍  status: approved", "gssoc23", "level1"]
```

**Review by priyankarpal — CHANGES_REQUESTED** [review]
try to cerate a separate PR & issue for project addition

File: .github/ISSUE_TEMPLATE/project-add.yml (hunk 1 above)
Comment (jibola2588) [reply .github/ISSUE_TEMPLATE/project-add.yml:4]: sir, I already made a PR for data addition which was merged yesterday

Comment (priyankarpal) [reply .github/ISSUE_TEMPLATE/project-add.yml:4]: ok, so could you create a separate & fresh PR? it will be good

Comment (priyankarpal) [reply .github/ISSUE_TEMPLATE/project-add.yml:4]: now I'm closing this PR

**Pull request closed without merging**